From e93fe39c1553da8bc81a08e4ab31698a24395a51 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Mon, 14 Oct 2013 13:35:16 +0200
Subject: [PATCH 09/18] qemu-char: fix documentation for telnet+wait socket flags

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1381757723-23134-9-git-send-email-kraxel@redhat.com>
Patchwork-id: 54854
O-Subject: [RHEL-7 qemu-kvm PATCH 08/15] qemu-char: fix documentation for telnet+wait socket flags
Bugzilla: 922010
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Reviewed-by: Laszlo Ersek <lersek@redhat.com>
Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
(cherry picked from commit ef993ba7bc631b57f9ea70a7f6fc8c0fcbdfa19d)
---
 qapi-schema.json | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 qapi-schema.json |    7 ++++---
 1 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/qapi-schema.json b/qapi-schema.json
index ffe0ed1..780c6fe 100644
--- a/qapi-schema.json
+++ b/qapi-schema.json
@@ -3213,10 +3213,11 @@
 # @addr: socket address to listen on (server=true)
 #        or connect to (server=false)
 # @server: #optional create server socket (default: true)
-# @wait: #optional wait for connect (not used for server
-#        sockets, default: false)
+# @wait: #optional wait for incoming connection on server
+#        sockets (default: false).
 # @nodelay: #optional set TCP_NODELAY socket option (default: false)
-# @telnet: #optional enable telnet protocol (default: false)
+# @telnet: #optional enable telnet protocol on server
+#          sockets (default: false)
 #
 # Since: 1.4
 ##
-- 
1.7.1

